float64 x
float64 y
float64 z
float64 relative_height
float64 x_des
float64 y_des
---
float64 z
bool got_terrain
---
float64 feedback
